src/main/java/org/opentripplanner/routing/core/RouteMatcher.java

```diff
@@ -184,4 +184,10 @@ public String toString() {
       routeNames
     );
   }
+
+  // TODO: 2022-08-22 I guess we want to go over to Matcher instead of using lists for unpreferredRoutes
+  // added this method so that we can map banned routes to McCostParamsBuilder
+  public HashSet<FeedScopedId> getAgencyAndRouteIds() {
```

Comment:
We should not use a matcher, upstream we moved to a collection of `FeesScopedIds`